orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -1,12 +1,19 @@
-<div class="list-by-status" ng-if="(todos | filter: {skipped_datetime: showSkipped?'!!':'!'}).length > 0">
-  <!-- <h4>{{title}}</h4> -->
-  <div class="todo"
-       ng-repeat="todo in todos | filter: {skipped_datetime: showSkipped?'!!':'!'}">
+<script type="text/ng-template" id="nodes_renderer.html">
```

Comment:
Can you record a loom video of dragging in action?

Reply:
I have disabled dragging todos right now.